config/environments/development.rb

```diff
@@ -72,4 +73,8 @@
 
   # Apply autocorrection by RuboCop to files generated by `bin/rails generate`.
   # config.generators.apply_rubocop_autocorrect_after_generate!
+
+  def get_oidc_config
```

Comment:
similar, if this is the default impl and noop is the production case, you reduce one repetition